src/ddmd/declaration.d

```diff
@@ -72,6 +72,113 @@ extern (C++) bool checkFrameAccess(Loc loc, Scope* sc, AggregateDeclaration ad,
     return result;
 }
 
+/***********************************************
```

Comment:
Didn't you mean to move `modifyFieldVar` to `src/ddmd/expressionsem.d`?

Reply:
no, the function is called just once from declaration.d. Don't know why it was in expression.

Reply:
I reviewed the pull-request commit by commit and I noticed that the [commit](https://github.com/dlang/dmd/pull/7251/commits/2df450d7f40f39ec2ad7397917caf11142c972d9) message of this change was "Move mofifyFieldVar to expressionsem and make it private", so it would be better to change the commit message to "Move mofifyFieldVar to declaration and make it private", in order to avoid confusion.